release.ps1

```diff
@@ -34,7 +34,25 @@ else
   exit 0;
 }
 
+$text = [string]::Format(((Get-Content $env:ACTION_PATH/octopus_configmap_old.yaml) -join "`n"),
```

Comment:
Move all the logic that you added to a separate file.

Reply:
Moved to separate file.